OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -65,6 +65,9 @@ public class ArmamentInfo : PausableConditionalTraitInfo, Requires<AttackBaseInf
 		[Desc("Condition to grant while reloading.")]
 		public readonly string ReloadingCondition = null;
 
+		[Desc("Whether the weapon is assumed to be reloaded if enough time has passed for a reload.")]
```

Comment:
```suggestion
		[Desc("Whether the weapon is assumed to be reloaded if enough time has passed between shots for a reload.")]
```
?
(To make clear under which conditions time has to pass for this to take effect.)